app/assets/stylesheets/views/_taxons.scss

```diff
@@ -69,33 +69,37 @@
   margin-top: $gutter * 2;
   @include taxon-element-styles;
 
-  ol {
+  .taxon-page__grid-wrapper {
     list-style-type: none;
     padding-bottom: $gutter-half;
-    @extend %grid-row;
+    font-size: 0; // Set so li elements don't wrap unnecessarily due to white space
 
-    li {
-      @include grid-column(1 / 3);
+    .taxon-page__grid-item {
+      @include core-19;
```

Comment:
Is this redundant? The headings and text in these elements already have font sizes assigned to them (which overrides this) although they are using element styling rather than explicit classnames, which would be good to fix. Again, might be outside the scope of this PR.